WikiDrips/WikiDoc.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  WikiDoc.swift
+//  WikiDrips
+//
+//  Created by Bart Cullimore on 4/3/17.
+//  Copyright © 2017 Bart Cullimore. All rights reserved.
+//
+
+import Foundation
+
+public class WikiDoc
+{
+    public let title = "Test Title String"
+    public let pubDate = Date.init()
```

Comment:
Usually not needed to explicitly call the init method: `public let pubDate = Date()`